Makefile.rules

```diff
@@ -924,8 +924,8 @@ coverage:
 ########################################################################
 
 PKG_BOOTSTRAP_URL = https://www.gap-system.org/pub/gap/gap4pkgs/
-PKG_MINIMAL = bootstrap-pkg-minimal.tar.gz
-PKG_FULL = bootstrap-pkg-full.tar.gz
+PKG_MINIMAL = packages-required-master.tar.gz
+PKG_FULL = packages-master.tar.gz
```

Comment:
Huh, wait -- that would then also affect the next stable release -- unless we change it back on the stable branch, or even change it in each release... Is that the plan?
In that case, it might be better to factor out the `master` bit into a separate variable, say, `PKG_BRANCH` and then do something like this (untested):
```make
PKG_BRANCH = master  # change to stable-X.Y in stable branch
PKG_MINIMAL = packages-required-$(PKG_BRANCH).tar.gz
PKG_FULL = packages-$(PKG_BRANCH).tar.gz
```

Reply:
Yes, I consider that change to be temporary, otherwise it will affect the next stable release of GAP 4.9.1 (not of GAP 4.8.9 if it will happen). The idea of $PKG_BRANCH$ seems good to me.  I will try it and then will do an "interactive rebase". 

Indeed, I've managed to do it in reverse order because I had all packages locally, and then pushed the whole branch with 4 commits.